sympy/physics/optics/polarization.py

```diff
@@ -0,0 +1,565 @@
+# -*- coding: utf-8 -*-
+r"""
+The module implements:
+
+- Jones vectors.
+
+- Stokes vectors.
+
+- Jones matrices.
+
+- Mueller matrices.
+
+Please see the description of the individual functions for further
+details and examples.
+
+Credits and Copyright
+~~~~~~~~~~~~~~~~~~~~~
+
+AUTHOR:
+
+- Oscar Gerardo Lazo Arjona (2017-06-18): added Wigner D matrices
+
+Copyright (C) 2019 Oscar Gerardo Lazo Arjona <algebraicamente@gmail.com>
```

Comment:
Remove these lines. The authorship of SymPy is reflected in the git history and in the AUTHORS file (which is generated automatically).

Reply:
Ok. There are similar lines in my previous contribution, though. I guess those should also be removed. Is that done in a separate PR?

Reply:
Yes, do that in a separate PR.

Reply:
I checked the AUTHORS file, and my name isn't there. Also, running bin/authors_update.py doesn't seem to update the file. How do I add my name?

Reply:
I'm not sure how the authors file is updated but I guess that you should be there given previous contributions...

Reply:
We update the AUTHORS periodically, and will at least do so before making a release. 

You can update it yourself it you want (but please do a separate PR). It requires running that script and then fixing any duplicates.